### ISSUE TITLE
forEach doesn't work if array item is null

### ISSUE THREAD

**Issue by BardachovArtem** [helpers/handlebars-helpers#313]
I use helper in this way
`{{# forEach allergens }}
    {{ tag }}{{# unless isLast }},{{/ unless }}
{{/ forEach }}`

but if `allergens = [{tag:32}, null]` I got an error `Cannot set property 'index' of null line`

**Comment by shanemac10** [comment]
Please see my open issue with [BigCommerce Stencil-CLI](https://github.com/bigcommerce/stencil-cli/issues/509), as this is causing server 500 errors and is any easy fix.

Thanks!

> My local fix was to add an empty array as a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), but you may choose to do a [fast-fail](https://en.wikipedia.org/wiki/Fail-fast)...
> `helpers.forEach = function(array=[], options) {`

**Comment by jonschlinkert** [comment]
Good catch... Since `null` won't cause the default to be used. Want to do a PR?

**Comment by iamtabrezkhan** [comment]
Hi @jonschlinkert 

Is this still open? I would be glad to do a PR to fix this.
